varinfo/generate_umm_var.py

```diff
@@ -37,6 +37,7 @@ def generate_collection_umm_var(
     auth_header: str,
     cmr_env: CmrEnvType = CMR_UAT,
     publish: bool = False,
+    config_file: Optional[str] = None,
```

Comment:
A preferred alternative here would be:

```
config_file: str | None = None
```

Because we also support Python 3.9, though, that means you need to add:

```
from __future__ import annotations
```

this goes right near the top of the file.

Reply:
Thanks Owen for the comments

Reply:
I switched that change 
    config_file: str | None = None 
back since it was failing for 3.9
Does the __future__ import basically make it not 3.9 anymore?

Reply:
The import of `__future__.annotations` will let the Python compiler use some specific newer features than that version had access to when it was released. [Here are the docs for the future part of the standard library](https://docs.python.org/3/library/__future__.html).